README.md

```diff
@@ -61,27 +61,17 @@ Install PostgreSQL / PostGIS and its dependencies,
 
 ### Create the database and role
 <pre>
-$ sudo su - postgres
-$ createdb 'hot_exports_dev'
-$ create role hot with password '<-password->'
+$ sudo -u postgres createuser -s -P hot
+$ sudo -u postgres createdb -O hot hot_exports_dev
 </pre>
 
 You might need to update the <code>pg_hba.conf</code> file to allow localhost connections via tcp/ip or
 allow trusted connections from localhost.
 
-Run <code>$ psql -h localhost -U hot -W hot_exports_dev</code>
-<pre>
-$ ALTER ROLE hot SUPERUSER;
-$ ALTER ROLE hot WITH LOGIN;
-$ GRANT ALL PRIVILEGES ON DATABASE hot_exports_dev TO hot;
-$ CREATE EXTENSION POSTGIS;
-$ CREATE EXTENSION HSTORE;
-</pre>
-
 Create the exports schema
 
 <pre>
-$ CREATE SCHEMA exports AUTHORIZATION hot;
+$ sudo -u postgres psql -d hot_exports_dev -c "CREATE SCHEMA exports AUTHORIZATION hot"
```

Comment:
This was to get around the need to modify `pg_hba.conf`, but since it has a password that would probably be better.

This looks like it works (it prompts for a password, which `sudo` might too, so it's an even trade-off):

```bash
psql -U hot -h localhost -d hot_exports_dev -c "CREATE SCHEMA exports AUTHORIZATION hot"
```

(`-h localhost` is needed to bypass the unix socket and prevent `psql: FATAL:  Peer authentication failed for user "hot"`)

Incidentally, I dropped the `exports` schema while developing (as a quick way to drop all of the tables) and `manage.py migrate` (and everything else) started using the `public` schema, so I think this can be skipped entirely. I'm probably going to run through the setup again soon so I'll check when I do.